CONTRIBUTING.rst

```diff
@@ -170,8 +170,13 @@ is a great way to get started if you'd like to make a contribution. Please note
 that our documentation files are in
 `ReStructuredText (.rst)
 <http://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html>`_
-format and format your pull request
-accordingly.
+format and format your pull request accordingly.
+
+To create a vignette for an EarthPy workflow, create a .py file that shows the
```

Comment:
## Creating Gallery Vignettes

We are using `sphinx-gallery` to create vignettes for EarthPy. To create a vignette for an EarthPy workflow, create a `.py` file that shows the

Reply:
add a link to the docs for sphinx gallery.